backend/apps/api/src/routes/upload.py

```
@@ -27,5 +31,23 @@ def create_presigned_put(req: UploadRequest):
+        image = Image(
+            user_id=user_id,
+            object_key=object_key,
+            filename=req.filename,
+            content_type=req.contentType,
+            size=req.size
+        )
+        db.add(image)
+        db.commit()
+        db.refresh(image)
+        print(f"✅ Image saved: ID={image.id}")
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)